allure-junit5-assert/README.MD

```diff
@@ -0,0 +1,14 @@
+Adding JUnit5-assert leads to java.lang.OutOfMemoryError: Java heap space
```

Comment:
may lead to...

Reply:
It would be a nice to add additional explanation:

1) Having a huge class path may lead to OOM, because AspectJ processes all the classes. Plus link to documentation https://www.eclipse.org/aspectj/doc/released/devguide/ltw.html
2) You can configure AspectJ to only process required classes
  a) package `io.qameta.allure` -> to process Allure classes
  b) package `org.junit` -> to process JUnit classes
  c) package  `com.company` -> to process your test classes

Link to documentation https://www.eclipse.org/aspectj/doc/released/devguide/ltw-configuration.html#configuring-load-time-weaving-with-aopxml-files

Then your example.